tests/end2end/test_single_operator_with_cntk_backend.py

```diff
@@ -384,9 +390,13 @@ def test_recursive_model(self):
         mapped1_2 = sub_model1(input1)
         mapped2_2 = sub_model2(input2)
         sub_sum = Add()([mapped1_2, mapped2_2])
-        model = Model(inputs=[input1, input2], outputs=sub_sum)
+        keras_model = Model(inputs=[input1, input2], output=sub_sum)
 
-        self._test_one_to_one_operator_core(model, [x, 2 * x])
+        coreml_model = coremltools.converters.keras.convert(keras_model)
```

Comment:
By calling _test_one_to_one_operator_core, we test conversion to from both of CoreML and Keras, this change means that we only test CoreML path now. I am not going to block you from checking in this important change but please do address my comments in another soon.

Reply:
sure, I'll immediately produce another PR to incorporate all the comments.

Reply:
Will do it in another PR